parm/config/gefs/config.base.emc.dyn

```diff
@@ -262,6 +262,10 @@ export OUTPUT_GRID="gaussian_grid"
 export WRITE_DOPOST=".true." # WRITE_DOPOST=true, use inline POST
 export WRITE_NSFLIP=".true."
 
+# Override normal post flat files for GEFS
+export FLTFILEGFS="${PARMgfs}/post/postxconfig-NT-GEFS.txt"
```

Comment:
@WalterKolczynski-NOAA I would like to know if these UPP control files post*GEFS.txt are deployed at ${PARMgfs}/post from an upp checkout revision UPP/parm during the checkout/build process?

Reply:
Yes. They are linked in from the UPP that is included when we check out UFS.

Reply:
@WalterKolczynski-NOAA Thanks for confirming.